### DISCUSSION TITLE
Is it possible to document a compiled Angular lib from node_modules?

### DISCUSSION THREAD

**Discussion by nategreen** [compodoc/compodoc#1119] [Q&A]
Hi 👋. I'm trying to use Storybook to show examples of [Kendo Angular components](https://www.telerik.com/kendo-angular-ui/components/), so we can develop theme CSS for them.

Storybook Angular uses Compodoc to generate `argTypes`, metadata about the components'/directives' properties. So my thinking was "hey, great, I can point Compodoc at the Kendo components in `node_modules` and then I don't have to manually enter/maintain the third-party components' properties in Storybook." Well, that doesn't seem to work. I've tried adding `node_modules/@progress/kendo-angular-buttons/dist/**/*` to `includes` in `tsconfig.json`, and when that didn't work I tried a bunch of other wacky stuff since I wasn't sure where the issue was.

I _think_ the issue is that Compodoc doesn't look at anything other than `.ts` and `.tsx` files. (Or that Typescript doesn't. I'm kind of a noob at this stuff.) That makes sense, because the normal usage of compodoc is documenting _my own_ code. But to me (being, again, a noob) it looks like most of the information Compodoc would need is in those `.js` and/or `.d.ts` files. 

**TL;DR:** Can I use Compodoc to target compiled Angular library code so it can provide metadata to Storybook?

Am I approaching this all wrong, or is this a worthy feature request? What would I even ask to be added/changed?

**Comment by murphomatic** [comment]
I'm looking to do exactly the same thing.  Did you ever get this sorted out?  Any input / direction would be awesome.

**Reply by nategreen** [reply]
I don't think I ever did. I am in a new role so I don't have access to that code anymore to be able to check.
